README.md

```diff
@@ -114,6 +113,13 @@ Js.log "Hello BuckleScript!"
 
 #### Use string interpolation
 
+```ml
+let () =
+  let world = {j|World|j} in
+  let hello_world = {j|Hello，$world|j} in
```

Comment:
variable name should be camleCase, but there's really no need for a variable here at all is it? You could just directly `Js.log` it.